spec/unit/spark_api/models/listing_spec.rb

```diff
@@ -351,6 +351,16 @@
           l.editable?(:PriceChange).should eq(true)
           l.editable?(:Photos).should eq(false)
         end
+
+        on_delete_it "should bulk delete listing photos" do
+          list_id = "20060725224713296297000000"
+          stub_api_get("/listings/#{list_id}", 'listings/with_photos.json', { :_expand => "Photos" })
+          l = Listing.find(list_id, :_expand => "Photos")
+          photo_id1 = l.photos[0].Id
+          photo_id2 = l.photos[1].Id
+          stub_api_delete("/listings/#{list_id}/photos/#{photo_id1},#{photo_id2}", 'success.json')
+          l.batch_photo_delete(photo_id1 + "," + photo_id2)
```

Comment:
Nothing is asserted in this test, which is a little bit odd. The example @wlmcewen suggested had an assertion that this returns true. That feels like a somewhat more useful test, as it confirms that something reasonable is being returned from the new method.

Looking at the new method compared to the existing methods around reordering, the bang versions of the reordering methods return true, while the new bang method returns the result of `connection.delete`.